dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/builder/TaskExecutionContextBuilder.java

```diff
@@ -69,7 +66,7 @@ public TaskExecutionContextBuilder buildTaskInstanceRelatedInfo(TaskInstance tas
     public TaskExecutionContextBuilder buildTaskDefinitionRelatedInfo(TaskDefinition taskDefinition) {
         taskExecutionContext.setTaskTimeout(Integer.MAX_VALUE);
         if (taskDefinition.getTimeoutFlag() == TimeoutFlag.OPEN) {
-            taskExecutionContext.setTaskTimeoutStrategy(taskDefinition.getTimeoutNotifyStrategy());
+            taskExecutionContext.setTaskTimeoutStrategy(org.apache.dolphinscheduler.spi.enums.TaskTimeoutStrategy.of(taskDefinition.getTimeoutNotifyStrategy().getCode()));
```

Comment:
Should it unified the `TaskTimeoutStrategy`?

Reply:
done.